src/hotspot/share/prims/jvmtiRedefineClasses.cpp

```diff
@@ -3694,6 +3679,19 @@ void VM_RedefineClasses::set_new_constant_pool(
       } // end for each local variable table entry
     } // end if there are local variable table entries
 
+    // Update constant pool indices in the method's method_parameters.
+    int mp_length = method->method_parameters_length();
+    if (mp_length > 0) {
+        MethodParametersElement* elem = method->method_parameters_start();
+        for (int j = 0; j < mp_length; j++) {
+            const int cp_index = elem[j].name_cp_index;
+            const int new_cp_index = find_new_index(cp_index);
+            if (new_cp_index != 0) {
+                elem[j].name_cp_index = new_cp_index;
+            }
+        }
```

Comment:
Indentation should be 2, not 4.

Reply:
Fixed. Also added explicit cast int -> u2